grafana-dashboard/README.md

```diff
@@ -0,0 +1,23 @@
+# Open Policy Agent Grafana dashboard
+
+![](images/dashboard-image-1.png)
+
+This folder hosts an example on the Grafana dashboard for Open Policy Agent.
+
+It visualizes most of the metrics recorded by Open Policy Agent as documented in 
+https://www.openpolicyagent.org/docs/latest/monitoring/
+
+The version of OPA that this is made for is 0.26.
+
+**Notes:** Before you use this dashboard, please mind the variables and the prometheus labels.
+It may not be applicable in your case or your K8s cluster may use another name than mine.
```

Comment:
“It” -> “They”. Also I’d avoid using “mine” here as this is the contrib repo and not a personal one, so it’s not clear to the reader who “me” is in this context. So I’d go with “use other names than those provided in this example” or something similar.